packages/workspace/src/utilities/project-graph-utils.ts

```diff
@@ -27,16 +28,42 @@ export function getSourceDirOfDependentProjects(
     );
   }
 
-  const dependencyNodeNames = findAllProjectNodeDependencies(
-    projectName,
-    projectGraph
-  );
-
-  return dependencyNodeNames.map(
+  const nodeNames = findAllProjectNodeDependencies(projectName, projectGraph);
+  return nodeNames.map(
     (nodeName) => projectGraph.nodes[nodeName].data.sourceRoot
   );
 }
 
+/**
+ * Finds the project node name by a file that lives within it's src root
+ * @param projRelativeDirPath directory path relative to the workspace root
+ * @param projectGraph
+ */
+export function getProjectNameFromDirPath(
+  projRelativeDirPath: string,
+  projectGraph = createProjectGraph()
+) {
+  let parentNodeName = null;
+  for (const [nodeName, node] of Object.entries(projectGraph.nodes)) {
+    const relativePath = relative(node.data.root, projRelativeDirPath);
+    const isMatch = relativePath && !relativePath.startsWith('..'); //
+    // && !isAbsolute(relativePath);
```

Comment:
What's the intention with this commented out code?

Reply:
forgot to remove 👍